developer/graphql-intro.md

```diff
@@ -0,0 +1,119 @@
+# Getting started with GraphQL
+
+## Install
+
+- Make sure you are on Reaction 1.10 and above.
+- Check your current Reaction version by running: `reaction -v`
+
+> Note: The GraphQL API is an experimental, alpha prototype. Do **not** use for production.
+
+## Authentication
+
+To access the GraphQL API, you need to be authenticated as an administrator via the app first:
+
+- Start Reaction as you normally would by running `reaction`
+- Open `http://localhost:3000`
+- Log in as an admin user
```

Comment:
It's actually not true that you need to be authenticated, and if you are authenticated, it does not have to be as admin. It might be better to start with `primaryShopId` as the first query because it doesn't require auth. This was added in 1.11, but it's probably better to say `1.11 and above` in the Install section anyway because things are more stable in that release.

So you can add a section above this, something like:

## Your first query

To try a simple query, go to http://localhost:3000/graphql-alpha in your browser, assuming you have Reaction running. In the query box, enter the following:

```js
{
  primaryShopId
}
```

... continue with "You should see a successful response" and the response similar to the existing "Your first query" section.

Then change the sentence at the top of the Authentication section to something like

"While some queries, such as `primaryShopId` do not require being logged in, other queries do require authentication and may either error or return different results depending on your authentication and authorization status. To authenticate, simply log in as you normally would in another browser tab in the same browser."

And finally change the existing "Your first query" heading to something like "Your first authenticated query"